aea/crypto/ethereum.py

```diff
@@ -95,14 +96,14 @@ def _load_private_key_from_path(self, file_name) -> Account:
         except IOError as e:        # pragma: no cover
             logger.exception(str(e))
 
-    def sign_transaction(self, tx_hash: bytes) -> bytes:
+    def sign_transaction(self, tx_hash: SignableMessage) -> bytes:
```

Comment:
relabel to sign_message

tx_hash should remain of type bytes; then inside this method convert to SingableMessage